airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -455,27 +455,24 @@ def test_nested_group(self, actual_connector_spec: ConnectorSpecification):
                 errors.append(f"Groups can only be defined on top level, is defined at {group_path}")
         self._fail_on_errors(errors)
 
-    def test_required_always_show(self, actual_connector_spec: ConnectorSpecification):
+    def test_display_type(self, actual_connector_spec: ConnectorSpecification):
```

Comment:
Why is this overriding the always show test case? Did it become irrelevant? I couldn't find anything explaining this in the PR description?

Reply:
Ah I thought I had included an explanation for this but apparently I didn't, sorry about that!

The reason I am removing `test_required_always_show` is because as of https://github.com/airbytehq/airbyte-platform-internal/pull/7763, we now support setting `always_true` on required fields, since it can now also be used to ensure that a field is always shown even if it is an OAuth output field, which doesn't have anything to do with the field being required or optional. So, this test is no longer relevant.